web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ViewSettingIT.java

```diff
@@ -0,0 +1,126 @@
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
+import edu.unc.lib.boxc.model.api.rdf.CdrView;
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import edu.unc.lib.boxc.operations.jms.viewSettings.ViewSettingRequest;
+import edu.unc.lib.boxc.web.services.rest.MvcTestHelpers;
+import edu.unc.lib.boxc.web.services.rest.exceptions.RestResponseEntityExceptionHandler;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.Statement;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class ViewSettingIT {
+    @InjectMocks
+    private ViewSettingController controller;
+    @Mock
+    private AccessControlService accessControlService;
+    @Mock
+    private RepositoryObjectLoader repositoryObjectLoader;
+    @Mock
+    private FileObject fileObject;
+    @Mock
+    private WorkObject workObject;
+    @Mock
+    private Resource resource;
+    @Mock
+    private Statement stmt;
+    private MockMvc mockMvc;
+    private AutoCloseable closeable;
+    private final static String USERNAME = "test_user";
+    private final static AccessGroupSet GROUPS = new AccessGroupSetImpl("adminGroup");
+    private static final String OBJECT_ID = "f277bb38-272c-471c-a28a-9887a1328a1f";
+    private static final PID OBJECT_PID = PIDs.get(OBJECT_ID);
+
+    @BeforeEach
+    public void setup() {
+        closeable = openMocks(this);
+        mockMvc = MockMvcBuilders.standaloneSetup(controller)
+                .setControllerAdvice(new RestResponseEntityExceptionHandler())
+                .build();
+        GroupsThreadStore.storeUsername(USERNAME);
+        GroupsThreadStore.storeGroups(GROUPS);
+        when(repositoryObjectLoader.getRepositoryObject(eq(PIDs.get(OBJECT_ID)))).thenReturn(workObject);
+        when(workObject.getResource()).thenReturn(resource);
+    }
+
+    @AfterEach
+    void closeService() throws Exception {
+        closeable.close();
+    }
+
+    @Test
+    public void testGetViewSetting() throws Exception {
+        var paged = ViewSettingRequest.ViewBehavior.PAGED.toString();
```

Comment:
Just wanted to note that this will set the value of the property to "PAGED", while the UI code currently has "paged" as the form input values. It's not going to cause any issues in the tests, but we'll have to settle on a value at some point

Reply:
Oh so should i just use the string `paged` in the tests?

Reply:
nvm i just used a string

Reply:
at some point we could add an extra method to the enum to get the lower case form, but this is good for now